doc/learning_godot.md

```diff
@@ -0,0 +1,21 @@
+Learning Godot
+==============
+
+Godot is the open source game engine that Thrive is built with.
+Godot resources, such as .tscn files, need to be modified
+in its editor.
+
+If you have never worked with Godot before it is recommended you
+start with the official documentation, specifically the [step by step section](https://docs.godotengine.org/en/stable/getting_started/step_by_step/intro_to_the_editor_interface.html)
+which goes over the basics of how the engine works. Additionally you
+should look at the [C# scripting section](https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_basics.html)
+since that is the language used by Thrive.
+
+The [Class Reference](https://docs.godotengine.org/en/stable/classes/index.html)
+is a resource that should be used often by programmers, it contains information
+on all of the properties, methods and signals of each of Godot's nodes.
```

Comment:
I'd perhaps add that by using an IDE that works well with Godot, you can see the class documentation directly there while coding. I've found that very useful.